config/500-webhook.yaml

```diff
@@ -59,6 +59,8 @@ spec:
                   fieldPath: metadata.namespace
             - name: CONFIG_LOGGING_NAME
               value: config-logging
+            - name: METRICS_DOMAIN
+              value: knative.dev/eventing
```

Comment:
how will we know this is not knative eventing?

Reply:
We don't. serving does the same, according to my understanding. Do you see a problem with this?

Reply:
Serving does not have anything like this repo as extensions, so they have not had to think about or solve this.